ci/lint/04_install.sh

```diff
@@ -1,12 +1,14 @@
 #!/usr/bin/env bash
 #
-# Copyright (c) 2018-2022 The Bitcoin Core developers
+# Copyright (c) 2018-present The Bitcoin Core developers
 # Distributed under the MIT software license, see the accompanying
 # file COPYING or http://www.opensource.org/licenses/mit-license.php.
 
 export LC_ALL=C
 
-export PATH=$PWD/ci/retry:$PATH
+export CI_RETRY_EXE="/ci_retry --"
```

Comment:
Now that we're setting this explicitly, does the prefixing of `CI_RETRY_EXE` still make sense in every case?
If it does, would it make sense to use it for e.g. `${CI_RETRY_EXE} curl` (like in https://github.com/bitcoin/bitcoin/blob/master/ci/test/01_base_install.sh#L88) or `${CI_RETRY_EXE} cargo build` commands as well?